varats/setup.py

```diff
@@ -22,9 +22,10 @@
     tests_require=["pytest", "pytest-cov"],
     install_requires=[
         "argparse-utils>=1.2.0",
-        "benchbuild>=6.0.1",
-        "click>=7.0.0",
+        "benchbuild>=6.1.1",
```

Comment:
```suggestion
        "benchbuild>=6.2.0",
```
Following the requirements.txt this should be 6.2